synapse/lib/cell.py

```diff
@@ -1921,7 +1935,7 @@ async def addDrivePath(self, path, perm=None, reldir=s_drive.rootdir):
 
             info = {
                 'name': name,
-                'perm': perm,
+                'permissions': perm,
```

Comment:
The `drive:add` Nexus handler will have to account for consuming events which have `perm` and not `permission`; since a mirror could be consuming a stream of events that were created prior to this change.